crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -349,6 +371,18 @@ impl RealRibosome {
         Ok((instance, context_key))
     }
 
+    fn cranelift_fn() -> fn() -> Cranelift {
+        || {
```

Comment:
Not a big deal but should probably just be a vanilla function and not a closure?